libs/estdlib/src/gen_statem.erl

```diff
@@ -169,6 +169,8 @@ stop(ServerRef) ->
 
 %%-----------------------------------------------------------------------------
 %% @param   ServerRef a reference to the gen_statem acquired via start
+%% @param   Reason to supply for stopping
+%% @param   Timeout to wait for shutdown
```

Comment:
Not sure about this phrase/rendering as well.
```suggestion
%% @param Reason reason to supply for stopping
%% @param Timeout maximum time to wait for shutdown
```

Reply:
fixed